.github/workflows/markdownlint.yml

```diff
@@ -0,0 +1,25 @@
+name: markdownlint
+
+on:
+  push:
+    branches: [ main ]
+    paths:
+    - '**.md'
+  pull_request:
+    branches: [ main ]
+    paths:
+    - '**.md'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: check out code
+      uses: actions/checkout@v3.0.1
+
+    - name: install markdownlint-cli
+      run: sudo npm install -g markdownlint-cli
```

Comment:
This way we are installing always the latest version. The build may start failing when a new version of `markdownlint` is released. It is NOT a blocker for me but we may consider using
1. `sudo npm install markdownlint-cli` and add `package-lock.json` + `package.json` files. Then we can use dependabot to bump depedencies
2. using `ghcr.io/igorshubovych/markdownlint-cli:v0.31.1`. This would make it easier for anyone not having `npm` installed. However it would be harder to track the versioning. We would need to use a `Dockerfile` so that dependabot sees new versions and then we need to `docker build` + `docker run` instead of only `docker run`.

Reply:
1st option covered by db2570b4a007bb95967cf3f5a859fabea5af9fbd and edb41cfea09e3fa40ea21d5a3d3ee3b6518f3694